bootstrap-app/src/cmr/bootstrap/api/messages.clj

```diff
@@ -84,19 +83,19 @@
   [params result]
   (if (api-util/synchronous? params)
     (format "Processed %s system concepts for bulk indexing." result)
-    (str "Processing system concepts for bulk indexing.")))
+    "Processing system concepts for bulk indexing."))
 
 (defn index-concepts-by-id
   [params result]
   (if (api-util/synchronous? params)
     (format "Processed %s concepts for bulk indexing." result)
-    (str "Processing concepts for bulk indexing.")))
+    "Processing concepts for bulk indexing."))
 
 (defn delete-concepts-by-id
   [params result]
   (if (api-util/synchronous? params)
     (format "Processed %s conccepts for bulk deletion from indexes." result)
```

Comment:
While you didn't make the change please fix the conccepts spelling by removing a c: to concepts.